docs/api/modules/App-Configuration.md

```diff
@@ -170,6 +171,65 @@ In practice, this allows POST requests from given origins to return partially re
 * [Frank Lloyd Root's `appConfig`](../../../prod-sample/sample-modules/frank-lloyd-root/0.0.0/src/config.js)
 * In Practice: [Partial Rendering](../../../README.md#partial-rendering)
 
+## `pwa`
+**Module Type**
+* ✅ Root Module
+* 🚫 Child Module
+
+> ⚠️ The PWA feature is behind a feature flag `ONE_SERVICE_WORKER` that needs to be
+> set to "true" during runtime.
+
+The `pwa` directive is how we configure the service worker and various PWA features.
+There are three distinct service workers with their given purpose. To use
+the standard service worker, we can set the `serviceWorker` to true. If there
+was any failure caused by the service worker, we can use the recovery service
+worker with `recoveryMode` set to true if we wish to reset browser clients. In
+the event we want to purge the service worker and remove any existing service
+workers on browser clients, We can use the escape hatch worker by setting
+`escapeHatch` to true.
+
+There is a precedence to which of the three flags is respected and the order goes:
+
+```
+1. serviceWorker
+2. escapeHatch
+3. recoveryMode
+```
+
+What this means is, if we set `{ recoveryMode: true, escapeHatch: true }`, `escapeHatch`
+will be honored (and enabled) instead of `recoveryMode`. To enable either of these modes,
```

Comment:
maybe add a couple a scenario of when you would use recoveryMode and escapeHatch.

you would use `recoveryMode` when ...
if this happened you may want to use `escapeHatch`...

Reply:
I've added a few cases and where these different workers would best be used here 02acbc4